travis/setup_dependencies_common.sh

```diff
@@ -140,7 +140,7 @@ fi
 # We pin the version for conda as it's not the most stable package from
 # release to release. Add note here if version is pinned due to a bug upstream.
 if [[ -z $CONDA_VERSION ]]; then
-    CONDA_VERSION=4.5.12
+    CONDA_VERSION=4.7
```

Comment:
When using in a pinned file I think this needs to be `4.7.*` otherwise the solver tries to find something with the exact version "4.7" which it will never find.

Reply:
Makes sense; at some point soon we need to change all of the pinnings to ranges, e.g. `>=4.7,<4.8` because I think the `.*` style is going away at some point (There have been deprecation warnings in the logs about it).